config/routes.rb

```diff
@@ -106,7 +106,8 @@ def external_redirect(path_prefix, target)
 
     get "/publications(.:locale)", as: "publications", to: "publications#index", constraints: { locale: valid_locales_regex }
     get "/publications/:id(.:locale)", as: "publication", to: "_#_", constraints: { locale: valid_locales_regex }
-    get "/publications/:publication_id/:content_id" => "_#_", as: "publication_html_attachment"
+    get "/publications/:publication_id/:content_id" => "_#_", as: "publication_html_attachment_by_content_id"
+    get "/publications/:publication_id/:id" => "_#_", as: "publication_html_attachment"
```

Comment:
I'd drop the first commit and just do all the route editing in this final commit.

As above, I'd want to put a comment above each 'legacy' route, explaining when and where it will be removed. We should then add a 'remove old routes' acceptance criteria to the 2nd card.

Reply:
I _believe_ that two routes will continue to be necessary, since English attachments will continue to be identified by the `:id` not the `:content_id`. And if there's a way to make that work with only one route I think we can do so straight away, I don't think the time-limited effect (to be reverted) is relevant to the routes.